<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBAFilterParamDlg</class>
 <widget class="QWidget" name="OBAFilterParamDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OBAFilterParamDlg</string>
  </property>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>oba seed generation</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox1_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>571</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>threshold for scene complex detection</string>
     </property>
     <widget class="QTextEdit" name="seedSlopeTh">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="seedHeightTh">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label1_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>slope threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>height threshold</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox1_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>571</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>window size for seeds generation</string>
     </property>
     <widget class="QTextEdit" name="seedCellSize">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="maxObjSize">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label1_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>small seed cell size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>max object size</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>oba segmentation</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox2_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>threshold for OBA segmentation</string>
     </property>
     <widget class="QTextEdit" name="segSlopeTh">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="segHeightTh">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label2_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>slope threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>height threshold</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>filter points</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox3_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>571</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>threshold for filtering las points</string>
     </property>
     <widget class="QTextEdit" name="filteringSlopeTh">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label3_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>local height threshold</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="filteringHeightTh">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label3_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>local slope threshold</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label3_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dtm resolution factor</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="dtmFactor">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label3_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>object height threshold</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="objectHT">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
